### ISSUE TITLE
JSON unmarshal error in GetTriggers

### ISSUE THREAD

**Issue by antoineco** [nukosuke/go-zendesk#177]
`(*Client).GetTriggers(context.Background(), &zendesk.TriggerListOptions{})` returns the following error:
```
json: cannot unmarshal bool into Go struct field TriggerCondition.triggers.conditions.all.value of type string
```

The culprit seems to be the following built-in Trigger:
```json
    {
      "url": "https://<redacted>.zendesk.com/api/v2/triggers/360168954433.json",
      "id": 360168954433,
      "title": "Notify requester and CCs of comment update",
      "active": true,
      "updated_at": "2020-07-20T16:01:33Z",
      "created_at": "2020-06-29T21:35:59Z",
      "actions": [
        {
          "field": "notification_user",
          "value": [
            "requester_and_ccs",
            "[{{ticket.account}}] Re: {{ticket.title}}",
            "Your request ({{ticket.id}}) has been updated. To add additional comments, reply to this email.\n{{ticket.comments_formatted}}"
          ]
        }
      ],
      "conditions": {
        "all": [
          {
            "field": "update_type",
            "operator": "is",
            "value": "Change"
          },
          {
            "field": "comment_is_public",
            "operator": "is",
            "value": true
          }
        ],
        "any": []
      },
      "description": null,
      "position": 5,
      "raw_title": "Notify requester and CCs of comment update"
    }
```

I also see a few of these with `"value": null`

```json
    {
      "url": "https://<redacted>.zendesk.com/api/v2/triggers/360168954453.json",
      "id": 360168954453,
      "title": "Notify assignee of comment update",
      "active": true,
      "updated_at": "2020-07-20T16:01:33Z",
      "created_at": "2020-06-29T21:35:59Z",
      "actions": [
        {
          "field": "notification_user",
          "value": [
            "assignee_id",
            "[{{ticket.account}}] Re: {{ticket.title}}",
            "This ticket (#{{ticket.id}}) has been updated.\n\n{{ticket.comments_formatted}}"
          ]
        }
      ],
      "conditions": {
        "all": [
          {
            "field": "comment_is_public",
            "operator": "is",
            "value": "not_relevant"
          },
          {
            "field": "assignee_id",
            "operator": "is_not",
            "value": "current_user"
          },
          {
            "field": "assignee_id",
            "operator": "is_not",
            "value": "requester_id"
          },
          {
            "field": "assignee_id",
            "operator": "not_changed",
            "value": null
          },
          {
            "field": "status",
            "operator": "not_value_previous",
            "value": "solved"
          }
        ],
        "any": []
      },
      "description": null,
      "position": 7,
      "raw_title": "Notify assignee of comment update"
    }
```

**Comment by nukosuke** [comment]
Hi, @antoineco 

#178 has been released at [v0.7.7](https://github.com/nukosuke/go-zendesk/releases/tag/v0.7.7)
Thank you!

**Comment by antoineco** [comment]
Thanks @nukosuke 🙌
